src/constants.js

```diff
@@ -24,6 +24,7 @@ export const LOGIN_RESULT = {
   SUCCESS: 'SUCCESS',
   INVALID_PASSWORD: 'INVALID_PASSWORD',
   CHANGE_PASSWORD: 'CHANGE_PASSWORD',
+  BLOCKED_USER: 'BLOCKED_USER',
```

Comment:
@talsalmona sorry for being picky, I think that `USER_BLOCKED` works better as a login result in this case